### PR TITLE
delete all duplicate empty blanks

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <764524258@qq.com>

### CODE REVIEW

File: pkg/notes/document.go [hunk 1]
```diff
@@ -94,7 +94,7 @@ func CreateDocument(notes []*ReleaseNote) (*Document, error) {
 func RenderMarkdown(doc *Document, w io.Writer) error {
 	// we always want to render the document with SIGs in alphabetical order
 	sortedSIGs := []string{}
-	for sig, _ := range doc.SIGs {
```
Comment (ixdy) [inline pkg/notes/document.go]: this is not the same.

**Comment by daixiang0** [comment]
@ixdy hi, any input?

**Comment by daixiang0** [comment]
@ixdy @marpaia  hi, any input?

**Pull request merged**